cmd/ct-woodpecker/main.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"path"
+	"syscall"
+	"time"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/ct-woodpecker/monitor"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	// default -config value
+	configDefault = "test/config.json"
```

Comment:
Given we only use this in the `flag.String` in `main` why define a top-level const?

Reply:
No reason. I'll move it local to main. Thanks!

Reply:
Fixed in de6d3a8